interfaces/automated-testing/scd/scd.yaml

```diff
@@ -298,6 +304,9 @@ components:
         volumes:
           description: >-
             Nominal volumes, as would be reported by a USS's operational_intents endpoint.
+
+            For U-Space UAS flight authorisation requests, volumes are required by ANNEX IV
```

Comment:
Same comment as for `Priority`; uss_qualifier is populating this field, so I don't think we need to specify requirements on how it must be populated here.  And, `volumes` are required anyway per original line 290

Reply:
I agree. I will discard this PR then.